src/main/scala/org/constellation/gossip/validation/MessageValidationError.scala

```diff
@@ -0,0 +1,15 @@
+package org.constellation.gossip.validation
+
+import org.constellation.schema.Id
+
+trait MessageValidationError extends Throwable {}
```

Comment:
I think that `sealed trait` would be a safer alternative, especially if you patter-match later on on this trait.